apps/transformers/test/unit/transformations/arithmetic_add_test.exs

```diff
@@ -0,0 +1,168 @@
+defmodule Transformers.ArithmeticAddTest do
+  use ExUnit.Case
+
+  alias Transformers.ArithmeticAdd
+
+  describe "transform/2" do
```

Comment:
We could probably delete some of these tests now

Reply:
If you like. Nothing particularly stood out to me as being redundant.